ladybug/rerunner/src/main/java/org/frankframework/ladybug/SpringBusRerunner.java

```diff
@@ -81,6 +84,9 @@ public String rerun(String correlationId, Report originalReport, SecurityContext
 		Map<String, String> threadContext = new HashMap<>();
 		// Try with resource will make sure pipeLineSession is closed and all (possibly opened)
 		// streams are also closed and the generated report will not remain in progress
+
+		int i = 0;
+		Checkpoint checkpoint;
```

Comment:
Mijn vingers jeuken bij deze code om dat om te schrijven naar een for-loop die begint bij 1, en aan het eind een `break` doet ipv de index te verhogen naar `size()`... 😳

Reply:
Ja ik heb ook jeuk, vooral omdat in de LadybugDebugger code het er nog erger uit ziet. Ik hoop tzt dit samen te kunnen voegen!

Reply:
Mogen de session-key values in de `threadContext` map wel `null` zijn?